ql/src/test/results/clientpositive/insert_overwrite_local_directory_1.q.out

```diff
@@ -0,0 +1,1788 @@
+PREHOOK: query: insert overwrite local directory '../../data/files/local_src_table_1'
```

Comment:
No need this file.
You should run Qtest  `insert_overwrite_local_directory_1.q` by `TestBeeLineDriver` and then this will change `ql/src/test/results/clientpositive/beeline/insert_overwrite_local_directory_1.q.out` automatically

Reply:
I have fixed this.